misk/src/main/kotlin/misk/web/HttpCall.kt

```diff
@@ -11,13 +11,31 @@ import okhttp3.RequestBody
 import okio.BufferedSink
 import okio.BufferedSource
 
+/**
+ * Information about the socket on which a HTTP call arrived.
+ */
+data class SourceSocket (
```

Comment:
Rename to SocketAddress. This is definitely not a socket, but merely it’s identifier